include/docca/base-stage1.xsl

```diff
@@ -320,17 +320,21 @@
           <xsl:template mode="parameter-row" match="parameteritem">
             <tr>
               <td>
-                <code>
-                  <!-- ASSUMPTION: <parameternamelist> only ever has one <parametername> child -->
```

Comment:
@vinniefalco It turned out to be a bad assumption, but at least I felt sufficiently unsettled about it to document it! 😂

Reply:
lmao... :)